Games/baj.2m.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# <bitbar.title>BAJ</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Justin Honold</bitbar.author>
+# <bitbar.author.github>astrostl</bitbar.author.github>
+# <bitbar.desc>Checks for active turns at http://boiteajeux.net every two minutes</bitbar.desc>
+# <bitbar.image></bitbar.image>
+# <bitbar.dependencies></bitbar.dependencies>
+
+# INSTRUCTIONS: set baj_login=LOGIN and baj_password=PASSWORD in ~/.bitbar
+
+. ~/.bitbar
```

Comment:
I agree that variables in scripts are an anti pattern - hopefully matryer/bitbar#88 will solve this. Until then, I think we should avoid doing things which might conflict with future progress, and stick to a convention, even if it is not the most desirable one.